micropython/drivers/led/neopixel/neopixel.py

```diff
@@ -22,11 +28,17 @@ def __init__(self, pin, n, bpp=3, timing=1):
             else timing
         )
 
+    def _apply_brightness(self, v):
```

Comment:
Rename to `_b` (to save qstr usage). It's a private method, so better to add a comment than to waste bytes on the name. (Unfortunately this is par for the course with MicroPython... there are lots of ugly things we have to do, ideally in places the user doesn't see, in order to save code size. Every byte counts).